### PR TITLE
setup the service repository doc and wiki

### PR DESCRIPTION
**Pull request by fredbelotte**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,100 @@
 # housing.forecast.api://contributing
 
+  If you would like to contribute to housing.forecast.api please follow these guidelines to ensure
```
Comment (fredbelotte) [inline .github/CONTRIBUTING.md]: we need to work on formatting.
+ headings are important
+ check for typos

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,100 @@
...
+The Pull Requests can have the following *labels*:
++_*Commited*_: The Commited label should indicate that work is being planned on an issue. We may need
+more information / we are planning on working on an issue.
++_*Started*_: The Started label should indicate that work / code is ready to be done or be started.
++_*Finished*_: The Finished label should indicate that the bulk of the work is done and it is waiting
+for a review. After this label it is likely to move back to started.
++_*Delivered*_: The Delivered label indicates that work has been accepted into Kramer.
+***
+ ###Contributiors:
+
```
Comment (fredbelotte) [inline .github/CONTRIBUTING.md]: let's make the names the link themselves for a cleaner look.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,51 @@
 # housing.forecast.api://readme
 
+This is a project focusing on placing incoming batches in appropriate accomodations.
```
Comment (fredbelotte) [inline README.md]: we are actually forecasting, not implementing selection

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (mrodsky) [reply .github/CONTRIBUTING.md]: will do. thank you for your feedback

File: README.md [hunk 4]
```diff
@@ -1,5 +1,51 @@
 # housing.forecast.api://readme
 
+This is a project focusing on placing incoming batches in appropriate accomodations.  
+It will give an accurate picture of the incoming potential associates and the open  
+housing options and will be a useful tool to the housing department.
+Contributors to this Repo are likely to work on the API and 
+make RESTful calls to the data context.
+
+## get started
+
```
Comment (fredbelotte) [inline README.md]: let's make the language more direct like
+ install .NET SDK 2.1.401+
+ install Docker 18.03+

File: README.md [hunk 5]
```diff
@@ -1,5 +1,51 @@
...
+..+Git HIstory by Don Jayamanne
+..+GitLens by Eric Amodio
+..+markdownlint by David Anson
+
+## documentation
+
+For more information on getting startedplease see our wiki at [Wiki Link](https://github.com/revaturecloud/housing.forecast.api/wiki).
+
+## contributors
+
```
Comment (fredbelotte) [inline README.md]: we do not need 2 sets of contributor list. work together and decide on 1 spot.

**Comment by mrodsky** [comment]
I found a tool to preview markdown online that helped. I went ahead and changed the names to links as well. I will talk to Daniel about his contributor section.

File: README.md (hunk 5 above)
Comment (Dmcgeough) [reply README.md]: Michael now has the only set of contributors.

File: README.md (hunk 4 above)
Comment (Dmcgeough) [reply README.md]: Shortened get started instructions to three short phrases and then a list of extensions required.

**Comment by mrodsky** [comment]
addressed that issue and assimilated Kat's style

**Pull request merged**